tests/test_load_audio.py

```diff
@@ -17,58 +17,72 @@ def test_load_wav(benchmark):
 
 def test_load_au(benchmark):
     # read au file
+    soundfile = pytest.importorskip(  # noqa
+        "soundfile", reason="If soundfile is not installed, this test will fail"
+    )
     path = Path(signalworks.__file__).parents[1] / "data" / "test.au"
-    with pytest.raises(Exception):
-        wave = benchmark(Track.read, path)
-        assert np.any(wave.value > 0)
-        assert wave.fs == 44100
+    wave = benchmark(Track.read, path)
+    assert np.any(wave.value > 0)
+    assert wave.fs == 44100
 
 
 def test_load_TIMIT(benchmark):
     # read NIST file
+    soundfile = pytest.importorskip(  # noqa
+        "soundfile", reason="If soundfile is not installed, this test will fail"
+    )
     path = Path(signalworks.__file__).parents[1] / "data" / "test.WAV"
-    with pytest.raises(Exception):
-        wave = benchmark(Track.read, path)
-        assert np.any(wave.value > 0)
-        assert wave.fs == 16000
+    wave = benchmark(Track.read, path)
+    assert np.any(wave.value > 0)
+    assert wave.fs == 16000
 
 
 def test_load_nis(benchmark):
     # read NIST file
+    soundfile = pytest.importorskip(  # noqa
+        "soundfile", reason="If soundfile is not installed, this test will fail"
+    )
     path = Path(signalworks.__file__).parents[1] / "data" / "test.nis"
-    with pytest.raises(Exception):
-        wave = benchmark(Track.read, path)
-        assert np.any(wave.value > 0)
-        assert wave.fs == 16000
+    wave = benchmark(Track.read, path)
+    assert np.any(wave.value > 0)
+    assert wave.fs == 16000
 
 
 def test_load_wa1(benchmark):
     # read WA1 file
+    soundfile = pytest.importorskip(  # noqa
+        "soundfile", reason="If soundfile is not installed, this test will fail"
+    )
     path = Path(signalworks.__file__).parents[1] / "data" / "test.wa1"
-    with pytest.raises(Exception):
-        wave = benchmark(Track.read, path)
-        assert np.any(wave.value > 0)
-        assert wave.fs == 8000
+    wave = benchmark(Track.read, path)
+    assert np.any(wave.value > 0)
+    assert wave.fs == 8000
 
 
 def test_load_wa2(benchmark):
     # read WA2 file
     path = Path(signalworks.__file__).parents[1] / "data" / "test.wa2"
-    with pytest.raises(Exception):
-        wave = benchmark(Track.read, path)
-        assert np.any(wave.value > 0)
-        assert wave.fs == 8000
+    soundfile = pytest.importorskip(  # noqa
+        "soundfile", reason="If soundfile is not installed, this test will fail"
+    )
+    wave = benchmark(Track.read, path)
+    assert np.any(wave.value > 0)
+    assert wave.fs == 8000
 
 
 def test_load_wv1(benchmark):
     # read WA2 file
     path = Path(signalworks.__file__).parents[1] / "data" / "test.WV1"
-    with pytest.raises(Exception):
-        benchmark(Track.read, path)
+    soundfile = pytest.importorskip(  # noqa
+        "soundfile", reason="If soundfile is not installed, this test will fail"
+    )
+    benchmark(Track.read, path)
```

Comment:
Why are we benchmarking this read; should we not have similar assert statements as the tests earlier?